gazebo_plugins/include/gazebo_plugins/gazebo_ros_openni_kinect.h

```diff
@@ -115,7 +115,10 @@ namespace gazebo
     private: sensor_msgs::PointCloud2 point_cloud_msg_;
     private: sensor_msgs::Image depth_image_msg_;
 
+    /// \brief Minimum range of the point cloud
     private: double point_cloud_cutoff_;
```

Comment:
I think it makes the most sense to rename the member at the same time as the SDF parameter, and that we shouldn't just rename the parameter without a tic-toc.